packages/dev/core/src/AudioV2/abstractAudio/streamingSoundInstance.ts

```diff
@@ -0,0 +1,34 @@
+import { Observable } from "../../Misc/observable";
+import { _AbstractSoundInstance } from "./abstractSoundInstance";
+import type { IStreamingSoundOptions, StreamingSound } from "./streamingSound";
+
+/** @internal */
+export abstract class _StreamingSoundInstance extends _AbstractSoundInstance {
+    private _resolvePreloadedPromise: () => void;
+
+    /** @internal */
+    public readonly onReadyObservable = new Observable<_StreamingSoundInstance>();
```

Comment:
Who notifies this observable?

Reply:
the `_WebAudioStreamingSoundInstance` notifies it.